openghg/transform/flux/_edgar.py

```diff
@@ -301,21 +331,20 @@ def parse_edgar(
 
     edgar_attrs = edgar_ds.attrs
 
-    # After the data has been extracted and used from the unzipped netcdf
-    # file clean up and remove temporary directory and file.
-    if zipped:
-        temp_extract_folder.cleanup()
-
     # Check for "time" dimension and add if missing.
     flux_ndim = flux_values.ndim
     time_name = "time"
     if time_name in flux_da:
         time = flux_da[time_name].values
+        flux = flux_values  # TODO: this was missing... is this correct?? otherwise 'flux' might be undefined
```

Comment:
@rt17603 can you confirm that this addition is correct? Previously `flux` wasn't defined if `time_name in flux_da` is True, and `flux` is needed later. (I'm guessing this means that we haven't seen an EDGAR file with a `time` variable yet.)

Reply:
This seems fine to me (looking at the file history this code may have evolved and so this omission was missed).